wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/AnyMessage.kt

```diff
@@ -91,7 +93,7 @@ class AnyMessage(
       }
 
       override fun decode(reader: ProtoReader): AnyMessage {
-        var typeUrl = "square.github.io/wire/unknown" // TODO: what does protoc do?
+        var typeUrl = ""
```

Comment:
protoc does that.

Reply:
SGTM!

Reply:
(though it’s now clear we don’t have a test for this)